docs/agent/config/v3.mdx

```diff
@@ -0,0 +1,584 @@
+| [`policy.outbound.name`](/docs/http/traffic-policy/index.mdx)                            | String                                                                       | `policy` has been renamed to `traffic_policy` in agent version 3.14.0.                                                                                                                                                                                                                            |
+| [`policy.outbound.expressions`](/docs/http/traffic-policy/expressions/index.mdx)         | Array of Strings                                                             | `policy` has been renamed to `traffic_policy` in agent version 3.14.0.                                                                                                                                                                                                                            |
+| [`policy.outbound.actions.type`](/docs/http/traffic-policy/actions/index.mdx)            | String                                                                       | `policy` has been renamed to `traffic_policy` in agent version 3.14.0.                                                                                                                                                                                                                            |
+| [`policy.outbound.actions.config`](/docs/http/traffic-policy/actions/index.mdx)          | custom                                                                       | `policy` has been renamed to `traffic_policy` in agent version 3.14.0.                                                                                                                                                                                                                            |
+| `proto`                                                                                  | `http`                                                                       | The endpoint protocol name. This defines the type of endpoint you would like to start.                                                                                                                                                                                                            |
+| `proxy_proto`                                                                            | String                                                                       | The version of [PROXY protocol](https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt) to use with this endpoint, empty if not using. Example values are 1 or 2.                                                                                                                            |
```

Comment:
this isn't in the "every option" example. might be ok, just calling that out.

Reply:
Yeah, I think the "every option" example is still lagging behind a bit, good callout.